### PR TITLE
FizzBuzz Challenge

### PR DESCRIPTION
**Pull request by vfrascello**
Hello, Please Review my Program.

### CODE REVIEW

**Comment by vfrascello** [comment]
@derekquinn

File: IdeaProjects/FizzBuzz2/src/com/company/Main.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            case 0: System.out.println("BrendenShilling");
+                break;
+            case 3: case 6: case 9: case 12: System.out.println("Brenden"); break;
+
+            case 5: case 10: System.out.println("Shilling"); break;
+            default: System.out.println(i); break;
+        }
+    }
+    }
+}
```
Comment (derekquinn) [inline IdeaProjects/FizzBuzz2/src/com/company/Main.java:18]: This looks good. I was curious to see what your `System.out` message would be.

**Comment by derekquinn** [comment]
PS: You have to give me write access to allow me to merge. You can check out [this resource](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/access-permissions-on-github) to learn how to do that.

**Comment by derekquinn** [comment]
MERGE IT NEWB!!!!

**Pull request merged**